server/protocols/webadmin/webadmin-data/src/main/java/org/apache/james/webadmin/routes/DLPConfigurationRoutes.java

```diff
@@ -58,7 +58,10 @@
 import spark.Request;
 import spark.Service;
 
-@Api(tags = "DLPRules")
+@Api(tags = "DLPRules",
+    description = "DLP (stands for Data Lost Prevention) is supported by James. A DLP matcher will, on incoming emails," +
```

Comment:
missing space after last comma :)

Reply:
I prefere to use `@ApiModel` as `@Api.description` is deprecated